src/Conan.cmake

```
@@ -33,46 +33,52 @@ macro(run_conan)
     URL
     https://bincrafters.jfrog.io/artifactory/api/conan/public-conan)
 
-  # For multi configuration generators, like VS and XCode
-  if(NOT CMAKE_CONFIGURATION_TYPES)
-    message(STATUS "Single configuration build!")
-    set(LIST_OF_BUILD_TYPES ${CMAKE_BUILD_TYPE})
+  if(CONAN_EXPORTED)
+    # standard conan installation, in which deps will be defined in conanfile. It is not necessary to call conan again, as it is already running.
+    include(${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake)
```

This should check for the existence of the path and issue a proper error.
```suggestion
    if (EXISTS "${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake")
      include(${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake)
    else()
       message(FATAL_ERROR "Could not set up conan because "${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake" does not exist")
    endif()
```